l1-contracts/src/core/interfaces/messagebridge/IInbox.sol

```diff
@@ -10,12 +10,18 @@ import {DataStructures} from "../../libraries/DataStructures.sol";
  * @notice Lives on L1 and is used to pass messages into the rollup from L1.
  */
 interface IInbox {
-  event LeafInserted(uint256 indexed blockNumber, uint256 index, bytes32 value);
+  /**
+   * @notice Emitted when a message is sent
+   * @param blockNumber - The block number in which the message is included
```

Comment:
The blocknumber here should probably be more specific, e.g., pointing to the l2 block, just to not confuse the value with `block.number`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/5383/commits/65ae3411439be47a89d9c21bd48726d7a114b56d